deb/package.go

```diff
@@ -576,6 +621,9 @@ func (p *Package) LinkFromPool(publishedStorage aptly.PublishedStorage, packageP
 		}
 
 		relPath := filepath.Join("pool", component, poolDir)
```

Comment:
should we build `relPath` outside of `LinkFromPool` so that we don't have to pass all the params? 

e.g. `publish.go` could have had correct logic for it, or we can make a method of `Package` which returns correct "pool" path depending on package type?

Something like `LinkFromPool(publishedStorage, packagePool, relPath, force)` ?